### ISSUE TITLE
inspectContainer fails when using unixHttpHandler

### ISSUE THREAD

**Issue by thoaionline** [denibertovic/docker-hs#80]
`inspectContainer` is failing with 

```
Left (DockerClientDecodeError "Error in $.NetworkSettings.Ports: parsing [] failed, expected Array, but encountered Null")
```

Another call `createContainer` seems to work fine under the same `DockerT` context.

**Comment by thoaionline** [comment]
#68 looks like the cure for this, the field can be null when returned from the API.

**Comment by jprider63** [comment]
Fixed with #81. Thanks for the report!
